lib/src/store.dart

```diff
@@ -146,4 +147,25 @@ class Store extends Stream<Store> with Disposable {
       trigger();
     }));
   }
+
+  /// A convenience method for listening to an [action] and triggering
+  /// automatically once the callback for said action has completed.
+  ///
+  /// If [onAction] is provided, it will be called every time [action] is
+  /// dispatched. If [onAction] returns a [Future], [trigger] will not be
+  /// called until that future has resolved.
+  ///
+  /// If the `Store` has been disposed, this method throws a [StateError].
+  void onActionTrigger<T>(Action<T> action,
```

Comment:
@evanweible-wf @georgelesica-wf or anyone .. how are you feeling about the name of this method vs `triggerOnAction` ?

Reply:
I'm on the fence.. our convention has been to just add a v2 to the end of method names like this so that the evolution is clear (if I saw these two methods on an API autocomplete, I would probably be confused as to the difference), but I know that doesn't result in the prettiest names.

Reply:
hmm V2 .. that kinda sucks, but I like it better than 2 slightly different names for the reason you point out. I'll change it to v2.